### PR TITLE
EDU-724 Create multitrack media player

### PR DESCRIPTION
**Pull request by ahelmberger**
Closes https://educandu.atlassian.net/browse/EDU-724

### CODE REVIEW

File: src/plugins/multitrack-media/multitrack-media.less [hunk 1]
```diff
@@ -1,3 +1,13 @@
 .MultitrackMediaDisplay {}
 
+.MultitrackMediaDisplay-content {
+  margin: 0 auto;
+}
+
+.MultitrackMediaDisplay-errorMessage {
+  font-weight: bold;
```
Comment (adrianaluchian) [inline src/plugins/multitrack-media/multitrack-media.less:8]: From my side I think we can drop this, makes the UI a bit too aggressive 🤔

Comment (ahelmberger) [reply src/plugins/multitrack-media/multitrack-media.less]: True, especially as this is kind of the initial default view. Dropped.

**Comment by adrianaluchian** [comment]
In edit mode, when we switch `sourceType` the `sourceUrl` is also reset, but the mixer still shows the old data

**Comment by adrianaluchian** [comment]
Such a cool and fun plugin 😍 Good job!

**Comment by adrianaluchian** [comment]
Small UI suggestion, if you feel it would be better, would be to write the times on the right hand side of the bar (as they represent the length, they might be more intuitive if placed there):

Screen Shot 2022-08-24 at 15 32 36

Not sure about the case of overflow to the right, if times should overflow a bit too:
Screen Shot 2022-08-24 at 15 31 57

**Comment by ahelmberger** [comment]
> In edit mode, when we switch `sourceType` the `sourceUrl` is also reset, but the mixer still shows the old data

I'd like to fix this (and other issues in this panel) with https://educandu.atlassian.net/browse/EDU-761 as I think we might want to change the way the track lengths are determined there anyway.

**Comment by ahelmberger** [comment]
> Such a cool and fun plugin heart_eyes Good job!

Thanks a lot :smile:

**Review by adrianaluchian — APPROVED** [review]

**Pull request merged**